packages/teleport-project-generator/src/index.ts

```diff
@@ -634,6 +635,7 @@ export class ProjectGenerator implements ProjectGeneratorType {
       collectedDevDependencies = { ...collectedDevDependencies, ...runAfterResult.devDependencies }
       inMemoryFilesMap = runAfterResult.files
     } catch (e) {
+      console.trace(e)
```

Comment:
we need this in the merge?

Reply:
Yes, right now. It is encapsulating the actual origin of error. We do have this already for the `runBefore` errors. Just added for `runAfter` too